### ISSUE TITLE
Is there a way to do a exact match？

### ISSUE THREAD

**Issue by yuchen-lea** [alphapapa/helm-org-rifle#73]
When searching string contains spaces, sometimes I want to do a exact match. This is a common case when searching book title. Is there a way to do a exact match？

**Comment by alphapapa** [comment]
No, but you can use `org-ql` to do that, e.g. with `helm-org-ql`.
